scripts/main.py

```diff
@@ -19,8 +30,19 @@ def main(args=None):
 
     if ns.infile[0] is not None:
         commodity_dictionary = cd.build_commod_dictionary()
-        facility_dictionary = pi.parse_input(ns.infile[0],
+        (facility_dict_in,
+         facility_dict_out) = pi.parse_input(ns.infile[0],
                                              commodity_dictionary)
+        (G, pathways) = ap.conduct_apa(facility_dict_in,
+                                       facility_dict_out)
+        # pa.print_graph_parameters(G, pathways)
+
+        # if ns.draw:
+        #     plt = pa.draw_graph(G)
+        #     plt.show()
+        # if ns.pickle:
+        #     nx.write_gpickle(G, ns.picklefile)
```

Comment:
do you need to keep those comments ?

Reply:
Sloppy on my part, its a holdover from when this PR included pathway analysis stuff. I guess for cleanliness I should remove the text entirely and then bring it back in a future PR where that code will actually do something